### PR TITLE
Implement Aboutus and Team UI in ReactJS

### PR DESCRIPTION
**Pull request by meyfleur**
Implement Aboutus and Team UI in ReactJS #75

## Description
updated styles according to [Design](https://www.figma.com/file/Mz0oDyKzMDCMhg2RapwbWFub/Ignitus?node-id=95%3A47):
updated some colors
override bootstrap base font-family and body color

added avatar pictures for Our Team page and background image
add team component with /Constants.js including:

`const AVATARS = [{
  title: 'Grant P. Her',
  path: '/images/about/avatar-grant.png',
  position: 'Product Manager',
},
{
  title: 'Melanie Caruso',
  position: 'Marketing Specialist',
  path: '/images/about/avatar-melanie.png',
},
{
  title: 'Nelson Bailey',
  position: 'Founder',
  path: '/images/about/avatar-nelson.png',
},`

would be great to replace the path and upload all avatar images somewhere to https://storage.googleapis.com/ignitus/

## Screenshots (if appropriate):
screencapture-localhost-3000-aboutus-2018-11-01-15_33_38

### CODE REVIEW

**Comment by divyanshu-rawat** [comment]
@meyfleur Thanks a lot for the PR 👍 .
For serving static Images S3 is fine, but we are already relying on GCP buckets for the same.
So, I will put all the images of about us page to the GCP bucket.

File: src/shared/styles/ignitus-utilities.scss [hunk 1]
```diff
@@ -1,6 +1,17 @@
 //Ignitus utility classes
 
-// text
-.align {
-  text-align: center;
```
[the remark this answers is not in the document]
Comment (divyanshu-rawat) [reply src/shared/styles/ignitus-utilities.scss]: @meyfleur can you please update the PR by adding that class then I will merge it. 👍 
Here is the generated preview - https://deploy-preview-159--peaceful-nightingale-55a1b7.netlify.com/

😃

[the remark this answers is not in the document]
Comment (meyfleur) [reply src/shared/styles/ignitus-utilities.scss]: done :)

[the remark this answers is not in the document]
Comment (divyanshu-rawat) [reply src/shared/styles/ignitus-utilities.scss]: Thank you so much 👍 😃

[the remark this answers is not in the document]
Comment (meyfleur) [reply src/shared/styles/ignitus-utilities.scss]: how do i fix the failed travis builds? maybe you can help :D

[the remark this answers is not in the document]
Comment (divyanshu-rawat) [reply src/shared/styles/ignitus-utilities.scss]: @meyfleur although, build didn't affected the Heroku pipeline your changes are deployed live on www.ignitus.org I will take care of fixing the Travis it's is beacuse package.json and lock file are not in sync not a big concern.

**Pull request merged**